lib/tasks/annual_fund_impact_metrics_activities.rake

```diff
@@ -12,6 +12,7 @@ namespace :activities do
         .joins(:organisation)
         .includes(:organisation, :actuals)
         .where.not(programme_status: ["delivery", "agreement_in_place", "open_for_applications", "stopped", "planned"])
+        .where.not(level: "fund")
```

Comment:
Is it worth mentioning in the commit that you haven't combined these two `where.not` as that would result in funds outside of those programme statuses being included and those programme statuses being included where they aren't funds (presumably)? Or is that obvious 😅

Reply:
This is also captured in the specs by happy coincidence! I think that should also be sufficient.